src/renderers/shared/fiber/ReactFiberScheduler.js

```diff
@@ -753,27 +753,31 @@ module.exports = function<T, P, I, TI, C>(config : HostConfig<T, P, I, TI, C>) {
       priorityLevel = TaskPriority;
     }
```

Comment:
What does this commit fix? Is there a test that was failing before? Is this a performance optimization?